rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -10395,10 +10396,8 @@ void opcode_0381(const ScriptArguments& args, const ScriptObject object, ScriptV
     @arg arg2 Boolean true/false
 */
 void opcode_0382(const ScriptArguments& args, const ScriptObject object, const ScriptBoolean arg2) {
-    RW_UNIMPLEMENTED_OPCODE(0x0382);
-    RW_UNUSED(object);
-    RW_UNUSED(arg2);
     RW_UNUSED(args);
+	object->setStatic(!arg2);
```

Comment:
The description of this opcode seems to be that it turns collision detection on and off, not change static status.

Reply:
You right. Will fix.